src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -170,6 +172,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
     [OuterLoop]
     public static void SendTimeout_Zero_Throws_TimeoutException_Immediately()
     {
+        TimeSpan serviceOperationTimeout = TimeSpan.FromMilliseconds(10000);
         BasicHttpBinding binding = new BasicHttpBinding();
```

Comment:
We are expecting it to timeout after 5 seconds, it "should" just take a few more milliseconds to get back to test code. The high-end of the range we check for should be equal to or less than the timeout we use for the service operation. When the bug is fixed and the test working again the service won't remain tied-up any longer than the SendTimeout because all the connections get shut-down by HttpClient.